### DISCUSSION TITLE
Windows 10 Start Menu Style does not show Store apps

### DISCUSSION THREAD

**Discussion by arcyc** [valinet/ExplorerPatcher#1372] [Q&A]
Properties > Start menu > Start menu style: Windows 10 causes Store apps to no longer show in the Start menu.  I can repro on a clean install of Windows 11 with default ExplorerPatcher settings.

**Comment by zetgzet** [comment]
I reported this problem via email twice. No response. After installing windows version 22H2, applications from the Windows Store in MenuStart are no longer visible when selecting the Windows10 StartMenu style, after returning to the Windows11 style, applications are visible as usual.

**Reply by zetgzet** [reply]
note - The Windows Store application also becomes "invisible".

**Comment by arcyc** [comment]
Looks like this is a known issue: https://github.com/valinet/ExplorerPatcher/issues/1108
